src/js/components/modal/modalCard.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { FunctionComponent } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 
-import PrintContent from '../mapWidgets/widgetContent/printContent';
-import ShareContent from '../mapWidgets/widgetContent/shareContent';
-import PenContent from '../mapWidgets/widgetContent/penContent';
-import SearchContent from '../mapWidgets/widgetContent/searchContent';
+import PrintContent from 'src/js/components/mapWidgets/widgetContent/printContent';
+import ShareContent from 'src/js/components/mapWidgets/widgetContent/shareContent';
+import PenContent from 'src/js/components/mapWidgets/widgetContent/penContent';
+import SearchContent from 'src/js/components/mapWidgets/widgetContent/searchContent';
+import CoordinatesForm from 'src/js/components/mapWidgets/widgetContent/coordinatesForm';
 
```

Comment:
I think all these imports can be `js/components....`, dont think we need to specify `src`. Let's confirm! I may be off base.

Reply:
Confirmed and implemented feedback @vaidotasp - thanks!